bundles/org.eclipse.passage.lic.net/src/org/eclipse/passage/lic/net/mail/api/LicensingMails.java

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright (c) 2019 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     ArSysOp - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.lic.net.mail.api;
+
+import org.eclipse.passage.lic.equinox.LicensingEquinox;
+import org.eclipse.passage.lic.net.mail.LicensingMail;
+
+/**
+ * @since 0.7
+ */
+public class LicensingMails {
```

Comment:
Let's try to avoid this approach: the service user should decide how to obtain an instance